libs/partners/ollama/Makefile

```diff
@@ -0,0 +1,62 @@
+.PHONY: all format lint test tests integration_tests docker_tests help extended_tests
+
+# Default target executed when no arguments are given to make.
+all: help
+
+# Define a variable for the test file path.
+TEST_FILE ?= tests/unit_tests/
+integration_test integration_tests: TEST_FILE = tests/integration_tests/
```

Comment:
solid chance we end up removing this until we split this into a separate repo for custom ci (need to spin up a tool calling ollama service for it)